src/win32.ts

```diff
@@ -31,39 +19,32 @@ export class WindowsApp extends App {
     return path.join(this.stagingPath, this.newElectronName);
   }
 
-  generateRceditOptionsSansIcon(): RceditOptions {
-    const win32metadata: Options['win32metadata'] = {
+  generateReseditOptionsSansIcon(): ExeMetadata {
+    const win32Metadata: Options['win32metadata'] = {
       FileDescription: this.opts.name,
       InternalName: this.opts.name,
       OriginalFilename: this.newElectronName,
       ProductName: this.opts.name,
       ...this.opts.win32metadata,
     };
 
-    const rcOpts: RceditOptions = { 'version-string': win32metadata };
-
-    if (this.opts.appVersion) {
-      rcOpts['product-version'] = rcOpts['file-version'] = this.opts.appVersion;
-    }
-
-    if (this.opts.buildVersion) {
-      rcOpts['file-version'] = this.opts.buildVersion;
-    }
-
-    if (this.opts.appCopyright) {
-      rcOpts['version-string']!.LegalCopyright = this.opts.appCopyright;
-    }
-
-    const manifestProperties = ['application-manifest', 'requested-execution-level'];
-    for (const manifestProperty of manifestProperties) {
-      if (win32metadata[manifestProperty as keyof typeof win32metadata]) {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        rcOpts[manifestProperty] = win32metadata[manifestProperty];
-      }
-    }
+    return {
+      productVersion: this.opts.appVersion,
+      fileVersion: this.opts.buildVersion || this.opts.appVersion,
+      legalCopyright: this.opts.appCopyright,
+      productName: this.opts.name,
+      win32Metadata,
+    };
 
-    return rcOpts;
+    // TODO
```

Comment:
This might be a breaking change

Reply:
Oh I implemented support for these, just didn't delete the commented out code 😅 Not breaking